auction_seller_invoice/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Auction Seller Invoice",
+    "category": "Auction",
+    "version": "15.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "license": "AGPL-3",
+    "depends": [
+        "auction_seller",
+        "account",
+        "account_commission",
```

Comment:
```suggestion
        "account",
        "account_commission",
        "auction_seller",
```
Please put the modules in the following order:
1. Odoo standard modules in alphabetical order
2. Others in alphabetical order